src/validation/loginValidation.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+import Joi from '@hapi/joi';
+
+export const signInValidationSchema = Joi.object().keys({
+  password: Joi.string().regex(/^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$/).required()
```

Comment:
why did you use regex, there are joi functions for this

Reply:
ok, thanks